lib/src/json_schema/json_schema.dart

```diff
@@ -1246,6 +1281,31 @@ class JsonSchema {
     ..addAll(_draft2019VocabMap)
     ..addAll(_draft2020VocabMap);
 
+  // configure the custom accessors for custom vocabularies.
+  Map<String, Map<String, SchemaPropertySetter>> _customVocabMap = Map();
+
+  // Hold values set by the custom accessors.
+  Map<String, CustomValidationResult Function(Object)> _customSetValue = Map();
+
+  // A wrapper.
+  SchemaPropertySetter _setCustomProperty(String property, KeywordProcessor processor) {
+    return (JsonSchema s, Object o) {
+      var obj = processor.setter(s, o);
+      CustomValidationResult Function(Object) fociwo = (Object instance) => processor.validator(obj, instance);
+      s._customSetValue[property] = fociwo;
```

Comment:
This is the point I gave up on giving things valuable names and just mashed my head into the keyboard.